### DISCUSSION TITLE
Venus 3.0 compatibility

### DISCUSSION THREAD

**Discussion by BobbyXXX** [Louisvdw/dbus-serialbattery#663] [Q&A] [answered]
Hi all, 

has anyone tried the driver with the beta of Venus is 3.0?

I would quite like to test the beta, but without the serial battery driver makes little sense to me 

Regards, Bobby

**Comment by wormsman** [comment, marked as answer]
Hi,

yes I running two JK BMS with the current nightly 20230518 and Venus OS 3.00~42 and it works great.

Regards
Nicolai

**Reply by BobbyXXX** [reply]
Great, good news 👍🏻

**Reply by mr-manuel** [reply]
It works also with driver version `v0.14.3` and Venus OS `> v3.00`, but there you will see a lot of additional empty fields.
